forms/serializers/form.py

```diff
@@ -244,6 +244,9 @@ def entry_generator(self, entries, sections=None, fields=None):
             yield from self.entry_generator(entries["sections"], sections=[entry for entry in entries["sections"]])
         if "fields" in entries:
             yield from self.entry_generator(entries["fields"], sections=sections, fields=[entry for entry in entries["fields"]])
+        if isinstance(entries, list):
```

Comment:
This could have been a fixup commit.

Reply:
https://blog.sebastian-daschner.com/entries/git-commit-fixup-autosquash#:~:text=Fixup%20commits%20produce%20commits%20that,and%20rebase%20the%20subsequent%20commits.